website/content/en/docs/config/network/user.md

```diff
@@ -43,4 +43,6 @@ DNS over tcp is rarely used. It is usually only used either when user explicitly
 
 During initial cloud-init bootstrap, `iptables` may not yet be installed. In that case the repo server is determined using the slirp DNS. After `iptables` has been installed, the forwarding rule is applied, switching over to the hostagent DNS.
 
-If `hostResolver.enabled` is false, then DNS servers can be configured manually in `lima.yaml` via the `dns` setting. If that list is empty, then Lima will either use the slirp DNS (on Linux), or the nameservers from the first host interface in service order that has an assigned IPv4 address (on macOS).
+If `hostResolver.enabled` is false (default for all drivers except `QEMU`), then DNS servers can be configured manually in `lima.yaml` via the `dns` setting. If that list is empty, then Lima will either use the slirp DNS (on Linux), or the nameservers from the first host interface in service order that has an assigned IPv4 address (on macOS).
+
+The host resolver is not needed with the `VZ` driver because the `gvisor-tap-vsock` implementation used by `usernet` has the same functionality built-in.
```

Comment:
nit

```suggestion
The host resolver is not needed with the `vz` driver because the `gvisor-tap-vsock` implementation used by `usernet` has the same functionality built-in.
```

or

```suggestion
The host resolver is not needed with the VZ driver because the `gvisor-tap-vsock` implementation used by `usernet` has the same functionality built-in.
```

Same for `QEMU`